tools/train.py

```diff
@@ -106,6 +110,9 @@ def main():
     if args.cfg_options is not None:
         cfg.merge_from_dict(args.cfg_options)
 
+    if args.auto_scale_lr:
```

Comment:
I feel that the configuration usage and args usage are inconsistent. It is best to unify all of them as `auto_scale_lr`

Reply:
I saw configuration like `--cfg-options` and `diff-seed` were using `-` and use `_` which were `args.cfg_options` and `args.diff_seed`. So I used `--auto-scale-lr` and `args.auto_scale_lr`. Do I still need to change `--auto-scale-lr` to `--auto_scale_lr` ? 😄

Reply:
Or do you mean the `enable_auto_scale_lr` should unify to `auto_scale_lr` ?

Reply:
> Or do you mean the `enable_auto_scale_lr` should unify to `auto_scale_lr` ?

I think we can try that. It is clear enough.

Reply:
Done